### PR TITLE
update package deps mental model to reflect 6.1

### PR DESCRIPTION
**Pull request by jensens**
This targets 6.1

### CODE REVIEW

File: coredev/packages-dependencies.md [hunk 1]
```diff
@@ -41,22 +41,35 @@ A base mental model for how Plone is organized in Plone 6 since alpha 4 is shown
...
+│ "Plone"                    |
+| The Integration of both    |
+| distributions in one       |
+| Release                    |
+├────────────────────────────┤
+| Distributions:             |
+| - plone.volto              |
+| - plone.classicui          |
+├────────────────────────────┤
+│ Core-Addons                │
```
Comment (stevepiercy) [inline coredev/packages-dependencies.md]:
```suggestion
│ Core add-ons:              │
```

File: coredev/packages-dependencies.md [hunk 2]
```diff
@@ -41,22 +41,35 @@ A base mental model for how Plone is organized in Plone 6 since alpha 4 is shown
...
+│ - plone.distribution       │
+│ - plone.app.exportimport   │
+│ - plone.app.discussion     │
+│ - plone.app.multilingual   │
+│ - plone.app.caching        │
 │ - plone.app.iterate        │
 │ - plone.app.update         │
 │                            │
 ├────────────────────────────┤
+│ Core-APIs                  │
```
Comment (stevepiercy) [inline coredev/packages-dependencies.md]:
```suggestion
│ Core APIs:                 │
```

File: coredev/packages-dependencies.md [hunk 3]
```diff
@@ -41,22 +41,35 @@ A base mental model for how Plone is organized in Plone 6 since alpha 4 is shown
...
+│ - plone.restapi            │
+│ - plone.api                │
+├────────────────────────────┤
 │                            │
 │     Products.CMFPlone      │
 │                            │
 ├────────────────────────────┤
 │                            │
-│ The space between          │
+│ The space between (core )  │
```
Comment (stevepiercy) [inline coredev/packages-dependencies.md:72]: "The space between (core )"? Between what two things?

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Reminder that docs in `/coredev` are not yet deployed in Production, and need to be migrated. See https://github.com/plone/buildout.coredev/issues/840#issuecomment-1441479383 for latest status.

Also I wonder if using Graphviz for the model instead of ASCII art might be visually more appealing, although more difficult to create if you don't know its syntax? I know that @MrTango created one simple Graphviz element in https://6.docs.plone.org/classic-ui/views.html. I don't have that knowledge beyond what is in the docs now.

**Comment by jensens** [comment]
> Also I wonder if using Graphviz for the model instead of ASCII art might be visually more appealing, although more difficult to create if you don't know its syntax?

I would prefer Mermaid - do we support it? Sphinx at least has a plugin I used in other projects already.

**Comment by stevepiercy** [comment]
> I would prefer Mermaid - do we support it?

We most certainly could! The only reason it is not currently supported is because no one has asked for it until now. 😁 

That `:zoom:` option is nifty.

https://pypi.org/project/sphinxcontrib-mermaid/

**Comment by stevepiercy** [comment]
@jensens I'm going to close this in favor of https://github.com/plone/documentation/pull/1671/, where you can insert a Mermaid diagram in docs at https://github.com/plone/documentation/pull/1671/files#diff-36199f263c1ec44ec26fc136ea22e39344bf44acc065e03e0fe26bc263d2f9f2 that will be published soon. The last step in that PR will be to purge the files at the root in `coredev`.

I already added support to the docs [`conf.py` in that PR](https://github.com/plone/documentation/pull/1671/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3) in anticipation. Please let me know. Thank you!

**Pull request closed without merging**